packages/workbox-webpack-plugin/src/generate-sw.js

```diff
@@ -34,12 +34,156 @@ const _generatedAssetNames = new Set();
  * @module workbox-webpack-plugin
  */
 class GenerateSW {
+  // eslint-disable-next-line jsdoc/newline-after-description
   /**
    * Creates an instance of GenerateSW.
    *
-   * @param {Object} [config] See the
-   * [configuration guide](https://developers.google.com/web/tools/workbox/modules/workbox-webpack-plugin#configuration)
-   * for all supported options and defaults.
+   * @param {Object} config The configuration to use.
+   *
+   * @param {Array<workbox-build~ManifestEntry>} [config.additionalManifestEntries]
```

Comment:
Does this need to be `module:workbox-build` in order for the link/reference to resolve?

Reply:
Yup.